.github/workflows/build.yml

```diff
@@ -7,7 +7,8 @@ on: [push, pull_request, workflow_dispatch]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    # NOTE: If we want to test with 3.6, we cannot run on ubuntu-latest anymore
+    runs-on: ubuntu-20.04
```

Comment:
Good to know. I think we probably want to stay on `ubuntu-latest`, so instead I'd suggest removing 3.6 from the strategy matrix

Reply:
Based on our conversation earlier, I've reversed this: sticking with `ubuntu-latest`, dropping Python 3.6.